events/notifications.go

```diff
@@ -13,8 +13,56 @@ import (
 	pkgNotification "github.com/flanksource/incident-commander/notification"
 	"github.com/flanksource/incident-commander/teams"
 	"github.com/google/uuid"
+	"gorm.io/gorm"
 )
 
+func NewNotificationConsumer(db *gorm.DB) EventConsumer {
+	return EventConsumer{
+		WatchEvents: []string{
+			EventNotificationSend, EventNotificationUpdate, EventNotificationDelete,
```

Comment:
I think event notification send should be its own consumer with 5 consumers - these can potentially be slow

Reply:
I think the event is duplicated now ?

Reply:
Added validation for that as well